master/internal/core.go

```diff
@@ -1245,6 +1245,7 @@ func (m *Master) Run(ctx context.Context, gRPCLogInitDone chan struct{}) error {
 		SSHRsaSize:            m.config.Security.SSH.RsaKeySize,
 		SegmentEnabled:        m.config.Telemetry.Enabled && m.config.Telemetry.SegmentMasterKey != "",
 		SegmentAPIKey:         m.config.Telemetry.SegmentMasterKey,
+		LogRetentionDays:      m.config.RetentionPolicy.LogRetentionDays,
```

Comment:
Am I understanding this correctly - we weren't reading the config value for log retention?

Reply:
I don't think we were ...

Reply:
Wow - nice catch!